specification/compute/resource-manager/Microsoft.Compute/stable/2020-05-01/disk.json

```diff
@@ -966,6 +966,275 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/diskAccesses/{diskAccessName}": {
+      "put": {
+        "tags": [
+          "DiskAccesses"
+        ],
+        "operationId": "DiskAccesses_CreateOrUpdate",
+        "description": "Creates or updates a diskAccess",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DiskAccessNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "diskAccess",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DiskAccess"
+            },
+            "description": "diskAccess object supplied in the body of the Put diskAccess operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DiskAccess"
+            }
+          },
+          "201": {
```

Comment:
Check with Cosmin if we should use 201 or 202

Reply:
Always 202

Reply:
Also: please check with Animesh: we should use the opportunity of a new API version to switch DiskEncryptionSets PUT to use 202 as well

Reply:
since this is an example for the purpose of API review, not including any changes specific to DES. On the final swagger update I'll make the suggested change after confirming with Animesh